### ISSUE TITLE
FUNCDEF macro no longer works in vasm m68k snapshot 2016-01-21

### ISSUE THREAD

**Issue by rkujawa** [Sakura-IT/SonnetAmiga#26]
After upgrading vasm, every call to `FUNCDEF` macro ends with the following error.

```
error 68 in line 2 of "FUNCDEF": repeatedly defined symbol <FUNC_CNT>
    called from line 12 of "exec/exec_lib.i"
    included from line 3 of "sonnet.s"
>FUNC_CNT    SET    FUNC_CNT-6  * Standard offset-6 bytes each
```

Mailed Frank about this.

**Comment by rkujawa** [comment]
The equate in macros should never be redefined. Fixed in commit 82f69f2ad632dd39589abef31180f6992eb9c2cf. It does assemble correctly on the server with vasm (source snapshot from two days ago).

**Comment by rkujawa** [comment]
Confirmed working.
